diff.py

```diff
@@ -2051,8 +2064,10 @@ def normalize_stack(row: str, arch: ArchSettings) -> str:
 
 
 def imm_matches_everything(row: str, arch: ArchSettings) -> bool:
-    # (this should probably be arch-specific)
-    return "(." in row
+    if arch.name == "ppc":
+        return any(str in row for str in ["...data", "@"])
```

Comment:
I'm worried `@` may occur in the middle of symbol names

Reply:
I did some searching on this, valid symbol names for asm, and also C++ mangling, and haven't seen anything yet that suggests a  @ can appear in a symbol.

even if it did... it would just make that symbol match any other symbol,

Reply:
MWCC will use `@` in the symbol names of thunks in C++, [here's an example from SMS](https://github.com/doldecomp/sms/blob/master/asm/Camera/sunmodel.s#L1007).

Reply:
hmm, shoot! I'll work on another option.

Reply:
okay my latest commit addresses the concern of @s appearing in symbol names.

Reply:
> MWCC will use @ in the symbol names of thunks in C++, [here's an example from SMS](https://github.com/doldecomp/sms/blob/master/asm/Camera/sunmodel.s#L1007).

That name also starts with a `@`, so middle of name vs start doesn't matter for it. (But perhaps the heuristic "starts with @ = matches anything" is wrong? or can we treat this symbol as matching anything?)

Reply:
> > MWCC will use @ in the symbol names of thunks in C++, [here's an example from SMS](https://github.com/doldecomp/sms/blob/master/asm/Camera/sunmodel.s#L1007).
> 
> That name also starts with a `@`, so middle of name vs start doesn't matter for it. (But perhaps the heuristic "starts with @ = matches anything" is wrong? or can we treat this symbol as matching anything?)

I think it would not be the end of the world if that symbol did match to everything(and by everything I mean any other symbol), however, In this PR I already have it fixed so that these symbols will not pass the imm_matches_everything test. 

So these symbols `"@32@__dt__9TSunModelFv"` will not be detected as a match with anything symbol